[DCOS-59715] Added docs for versions, limitations, and release notes (#33)
mesosphere/kudo-spark-operator: kudo-operator/docs/latest/limitations.md

Limitations
---

## Multi-instance installation
* Currently, multi-instance (multi-tenant) operator installation supports only a single instance per namespace to
allow Spark applications be launched in the namespace they've been submitted to. Multiple operator instances

> installed in the same namespace will be handling submissions in parallel which can potentially lead to race conditions

Reviewer: ```suggestion
installed in the same namespace run job submissions in parallel which can potentially lead to race conditions
```
